modules/distr-tracing-tempo-install-instance.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-instance_{context}"]
+= Installing a {TempoName} instance
+
+https://issues.redhat.com/browse/TRACING-3296
+
+Deploying the TempoStack
+
+You can use the OpenShift Container Platform web console to deploy the TempoStack.
+
+.Prerequisites
+
+* Supported Object Store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+Install the Tempo Operator:
+
+In the OpenShift Container Platform web console, click Operators → OperatorHub.
+
+Choose Tempo Operator from the list of available Operators, and click Install.
+
+Under Installation Mode, select All namespaces on the cluster.
+
+Under Installed Namespace, select openshift-operators-redhat.
+
+You must specify the openshift-operators-redhat namespace. The openshift-operators namespace might contain Community Operators, which are untrusted and might publish a metric with the same name as an OpenShift Container Platform metric, which would cause conflicts.
+
+Select an Approval Strategy.
+
+The Automatic strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
+
+The Manual strategy requires a user with appropriate credentials to approve the Operator update.
+
+Click Install.
+
+Verify that you installed the Tempo Operator. Visit the Operators → Installed Operators page and look for Tempo Operator.
+
+Ensure that Tempo Operator is listed with Status as Succeeded in all the projects.
+
+Create a Secret YAML file that uses the access_key_id and access_key_secret fields to specify your AWS credentials and bucketnames, endpoint and region to define the object storage location. For example:
+
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-test
+stringData:
+  endpoint: http://minio.minio.svc:9000
+  bucket: tempo
+  access_key_id: tempo
+  access_key_secret: supersecret
+type: Opaque
+----
+
+Create the TempOStack custom resource, this is a basic example:
+
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: simplest
+spec:
+  storage:
+    secret:
+      name: minio-test
+      type: s3
+  storageSize: 1Gi
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+----
+
+Apply the configuration:
+
+----
+oc apply -f tempo.yaml
+----
+
+After create the TempoStack CR you should see a some pods on the namespace, wait until the stack to stabilized.
```

Comment:
```suggestion
. Verify that are pods on the namespace when the `TempoStack` CR is created.

.  Wait for the stack to stabilize.
```

Reply:
After this step. We can add another step to get the Tempo query frontend route.

`To get the Tempo query frontend URL, run the below command.`
`oc get route`